submissions/guizo792/Main.java

```diff
@@ -0,0 +1,60 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Map<String, Double> map = new HashMap<>();
+
+        try (Stream<String> lines = Files.lines(Paths.get("input.txt"))) {
+            List<String> linesList = lines.collect(Collectors.toList());

```

Comment:
IMO this step is unnecessary and will create a list of 22G worth of data which can lear to an OOM. You can handle the stream directly